samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -138,7 +138,8 @@ void main(void)
 
 	gpio_pin_configure(gpio, PIN,
 			   GPIO_DIR_IN | GPIO_INT |
-			   GPIO_INT_EDGE | GPIO_INT_DOUBLE_EDGE);
+			   GPIO_INT_EDGE | GPIO_INT_DOUBLE_EDGE |
+			   GPIO_PUD_PULL_UP);
```

Comment:
If we enable the pull-up, it means that the default state is a `1` and and that pressing the button gives a `0`. I guess it's then possible to simplify the code by replacing `def_val` by `1`.

Reply:
I have used PIN_PUD referencing to SW0_GPIO_PIN_PUD instead.